electricpy/bode.py

```diff
@@ -20,9 +21,9 @@ def _sys_condition(system,feedback):
         den = system[1]
         # Convolve numerator or denominator as needed
         if (str(type(num)) == tuple):
-            num = convolve(num)        # Convolve terms in numerator
+            num = sig.convolve(num)        # Convolve terms in numerator
```

Comment:
This change cannot be accepted. I've defined a "local" `convolve` function which allows more than two items to be convolved in a single call. It's intentionally done this way to allow more flexibility in the convolution. There's ways this could be improved, but I don't think using `scipy.signal.convolve` is appropriate right here.